docs/custom-commands/conditions.md

```diff
@@ -0,0 +1,27 @@
+---
+id: Command-Conditions
```

Comment:
These shouldn't be separate pages, there should be a single page with clickable sections imo. Maybe a section called "Commands", which has a page called "Custom Commands"